src/tutorials/graphConfigs.ts

```diff
@@ -0,0 +1,175 @@
+.setType(TutorialStep.Type.Press)
+
+newTut.newTutStep("Managing Graph Configurations", "In this table we can view, edit, copy or delete existing graph configurations saved in this graph. <em>next to continue</em>", function(){return $('.column-actions').first()})
+.setAlternateHighlightTargetFunc(function(){return $('#bottomWindow .content')})
+
+newTut.newTutStep("Managing Graph Configurations", "Lets say we want another version of this config, but also be able to easily change the output file path. First, we must duplicate our existing graph configuration. <em>Click to duplicate our first configuration</em>", function(){return $('.btmWindowDuplicateBtn').first()})
+.setType(TutorialStep.Type.Press)
+
+newTut.newTutStep("Managing Graph Configurations", "Our configuration has been duplicated, lets add the additional field. <em>next to continue</em>", function(){return $('.bottomWindowHeader span')})
+.setAlternateHighlightTargetFunc(function(){return $('#bottomWindow .content')})
+
+newTut.newTutStep("Adding another field", "<em>Click on the File node to select it.</em>",  function(){return TutorialSystem.initiateFindGraphNodeIdByNodeName('File')})
+.setType(TutorialStep.Type.Condition)
+.setWaitType(TutorialStep.Wait.Element)
+.setConditionFunction(function(){return TutorialSystem.isRequestedNodeSelected('File')})
+.setPreFunction(function(eagle:Eagle){eagle.resetEditor()})
+.setBackPreFunction(function(eagle:Eagle){eagle.resetEditor()})
+
+newTut.newTutStep("Adding another field", "<em>Click to Switch to the node fields table</em>", function(){return $('#bottomTabParamsTableSwitcher')})
+.setType(TutorialStep.Type.Press)
+
+newTut.newTutStep("Adding another field", "<em>Click on the heart to flag the file path field as a graph configuration field.</em>", function(){return $('.column_DisplayText button').first()})
+.setType(TutorialStep.Type.Press)
+.setPreFunction(function(){setTimeout(function(){$('.column_DisplayText button').first().css('visibility','visible')},200)})
+.setWaitType(TutorialStep.Wait.Delay)
+.setDelayAmount(700)
+
+newTut.newTutStep("Adding another field", "Switch back to the graph configuration fields table <em>Click to Switch</em>", function(){return $('#bottomTabKeyParamsSwitcher')})
+.setType(TutorialStep.Type.Press)
+
+newTut.newTutStep("Adjusting Graph Configurations", "We can see that the FilePath field has been added to our graph configuration. Describe what it does, such as 'the output file path of the graph' <em>Type a description and press Enter</em>", function(){return $('.tableFieldCommentInput').eq(1)})
+.setType(TutorialStep.Type.Input)
+.setWaitType(TutorialStep.Wait.Delay)
+.setDelayAmount(200)
+
+newTut.newTutStep("Saving Graph Configurations", "Dont forget to save the new config</em>", function(){return $('#parameterTableSaveGraphConfig')})
```

Comment:
Missing apostrophe in "Don't"

Reply:
fixed. if that was my worst spelling mistake in all that, i did pretty well! haha